main-app/app/scripts/bingoTicket/bingoModel.js

```diff
@@ -20,14 +20,29 @@
                 }
             };
 
+            var BingoCell = function () {
+                this.bingoNumber = null;
+                this.marked = "unmarked";
+                this.counted = false;
+            };
+
+            var TicketInfo = function () {
+                this.row1Marked = 0;
+                this.row2Marked = 0;
+                this.row3Marked = 0;
+                this.numbersToWin = 5;
+            };
+
             var createBingoTicketGrid = function () {
                 me.ticketGrid = [
-                    [null, null,null,null,null,null,null,null,null],
-                    [null, null,null,null,null,null,null,null,null],
-                    [null, null,null,null,null,null,null,null,null]
+                    [new BingoCell(), new BingoCell(), new BingoCell(), new BingoCell(), new BingoCell(), new BingoCell(), new BingoCell(), new BingoCell(), new BingoCell()],
```

Comment:
The individual cells should be created within each row object.
Similarly the rows should be created inside the ticket object.